Source/SpeechToTextV1/Tests/SpeechToTextTests.swift

```diff
@@ -277,6 +277,57 @@ class SpeechToTextTests: XCTestCase {
         }
         waitForExpectations()
     }
+    
+    // MARK: - Transcribe Data with Smart Formatting
+    
+    func testTranscribeStockAnnouncementCustomWAV() {
+        transcribeDataCustomForNumbers("StockAnnouncement", withExtension: "wav", format: .WAV)
+    }
```

Comment:
The `testTranscribeStockAnnouncementCustomWAV()` and `transcribeDataCustomForNumbers(...)` functions seem coupled. The asserts in `transcribeDataCustomForNumbers(...)` check that the particular number $152.37 occurs in the audio transcription. So if we execute that function for a different audio file, it may fail because it doesn't contain the number $152.37.

To decouple these functions, it's probably easiest to combine them. I like the idea of having a helper function for testing numbers, but I don't expect us to have any other tests with numbers. So I think it's safe to combine them.

Alternatively, you could add a `substring: String` argument to `transcribeDataCustomForNumbers`:

```
transcribeDataCustomForNumbers("StockAnnouncement", withExtension: "wav", format: .WAV, substring: "$152.37")
```